### PR TITLE
chore: rename var to clarify meaning

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

File: midealocal/device.py [hunk 1]
```diff
@@ -94,10 +94,10 @@ def __init__(
         self._device_name = name
         self._device_id = device_id
         self._device_type = device_type
-        self._protocol = protocol
+        self._device_protocol_version = protocol
         self._model = model
         self._subtype = subtype
-        self._protocol_version: ProtocolVersion = ProtocolVersion.V1
+        self._message_protocol_version: ProtocolVersion = ProtocolVersion.V1
```
Comment (rokam) [inline midealocal/device.py:100]:
```suggestion
        self._message_protocol_version: ProtocolVersion = 0
```

Comment (chemelli74) [reply midealocal/device.py:100]: Will make the logic change in a different PR, this is just the rename one ;-)

File: midealocal/message.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def __init__(self) -> None:
         self._device_type: DeviceType = DeviceType.X00
         self._message_type: MessageType = MessageType.default
         self._body_type: BodyType = BodyType.X00
-        self._protocol_version: ProtocolVersion = ProtocolVersion.V1
+        self._message_protocol_version: ProtocolVersion = ProtocolVersion.V1
```
Comment (rokam) [inline midealocal/message.py:138]:
```suggestion
        self._message_protocol_version: ProtocolVersion = 0
```

Comment (chemelli74) [reply midealocal/message.py:138]: Only renaming in this PR, logic will be changed in an upcoming one

**Review by rokam — APPROVED** [review]
LGTM

**Pull request merged**